src/storage/sftp/storage.c

```diff
@@ -850,16 +905,28 @@ storageSftpNew(
 #endif // LIBSSH2_HOSTKEY_HASH_SHA256
 
             default:
-                THROW_FMT(
-                    ServiceError, "requested ssh2 hostkey hash type (%s) not available", strZ(strIdToStr(hostKeyHashType)));
+                THROW_FMT(ServiceError, "requested ssh2 hostkey hash type (%s) not available", strZ(strIdToStr(hostKeyHashType)));
                 break;
         }
 
-        const char *binaryFingerprint = libssh2_hostkey_hash(this->session, hashType);
+        const char *const binaryFingerprint = libssh2_hostkey_hash(this->session, hashType);
 
         if (binaryFingerprint == NULL)
             THROW_FMT(ServiceError, "libssh2 hostkey hash failed: libssh2 errno [%d]", libssh2_session_last_errno(this->session));
 
+        // Is known_hosts search enabled
```

Comment:
Will update this comment to
// Enable known_hosts search unless param.knownHostsFiles (--repo-sftp-known-hosts-files) is explicitly set to disable it. NULL results in searching a default known_hosts file list.

Reply:
Updated in aab080ae3528d998b46ab7b6905d1fb9e8083519